--- /usr/src/linux-2.4.24/drivers/net/wireless/Makefile	2004-02-10 22:38:17.000000000 -0800
+++ Makefile.wireless	2004-02-10 09:55:38.000000000 -0800
@@ -14,6 +14,9 @@
 # Things that need to export symbols
 export-objs	:= airo.o orinoco.o hermes.o
 
+subdir-$(CONFIG_ATHEROS)	+= ath
+subdir-$(CONFIG_ATHEROS_HAL)	+= _ath_hal
+
 obj-$(CONFIG_HERMES)		+= orinoco.o hermes.o
 obj-$(CONFIG_PCMCIA_HERMES)	+= orinoco_cs.o
 obj-$(CONFIG_APPLE_AIRPORT)	+= airport.o
@@ -24,4 +27,9 @@
 obj-$(CONFIG_AIRO)		+= airo.o
 obj-$(CONFIG_AIRO_CS)		+= airo_cs.o airo.o
 
+ifeq ($(CONFIG_ATHEROS),y)
+	obj-y += ath/ath.o
+	obj-y += _ath_hal/ath_hal.o
+endif
+
 include $(TOPDIR)/Rules.make
